scripts/cast-dssspell.sh

```diff
@@ -21,8 +21,9 @@ castSend() {
     set -e
     echo "cast send $*"
     ETH_GAS=$(cast estimate "$@")
```

Comment:
Let's just remove `ETH_GAS` here.
```suggestion
    ETH_GAS_LIMIT=$(cast estimate "$@")
```